vrt/ci.js

```diff
@@ -34,6 +34,11 @@ const octokit = Octokit({
 
 main().catch(handleError);
 
+process.on('unhandledRejection', function(err) {
+  console.log(err);
+  throw err;
+});
+
```

Comment:
does this remove the need for the `process.exit` and `handleError` code?

Reply:
yeah, I think so